### PR TITLE
Translate Notifications Configuration

### PR DESCRIPTION
**Pull request by cconard96**
Built on top of pending PRs to make it easier to track what was left to translate and update references to this documentation.
Commits specific to this PR:
2c9e0650a4baca60392ba5c186ea509926872fbf...0d743c6c9c5144b53c12bead5c23a5e97b44c857

### CODE REVIEW

**Comment by trasher** [comment]
Seems OK for the specific commits. Maybe you can rebase this one?

**Comment by cconard96** [comment]
Reset to current master and manually cherry-picked related commits back in.
There was some conflict with a previous notifications commit. I merged conflicting pages when needed and fixed some typos and other issues I found from the other commit.

ping @Rom1-B Do my changes from your commit look OK?

File: source/modules/configuration/notifications/template_example.rst [hunk 1]
```diff
@@ -22,25 +22,25 @@ Create template
...
 
 1. Go to **Setup > Notifications > Notification templates**
-2. Clic on **add** button
+2. Click on **Add** button
 3. Fill in fields
 
-   - **Name** = ``Generic tickets with timeline`` *for example*
-   - **Type** = ``Ticket``
-   - **Comments** = ``Generic template for tickets, using timeline`` *for example*
+   - **Name*: ``Generic tickets with timeline`` *for example*
```
Comment (Rom1-B) [inline source/modules/configuration/notifications/template_example.rst:28]:
```suggestion
   - **Name**: ``Generic tickets with timeline`` *for example*
```

File: source/modules/configuration/notifications/template_example.rst [hunk 2]
```diff
@@ -210,12 +210,12 @@ And we want to add a text under the URL that everyone can understand:
...
 1. In the **Default translation** we add the English text
 2. Remember to **Save**
-3. **Right-clic** in "Email HTML body" field > **Source code**
+3. **Right-click** in "Email HTML body" field > **Source code**
 4. Select all the code to **copy** it
 5. Go back to the template header by clicking on the **Notification templates** value at the top
 6. In **Template translations** tab click on **Add a new translation** button
-7. Select Language = **Français**
-8. **Right-clic** in "Email HTML body" field > **Source code**
+7. Select Language = **French**
```
Comment (Rom1-B) [inline source/modules/configuration/notifications/template_example.rst:217]: I think it's "Français" in the drop-down list?
```suggestion
7. Select Language = **Français**
```

Everything seems OK to me

**Review by Rom1-B — APPROVED** [review]

File: source/modules/configuration/notifications/template_example.rst (hunk 2 above)
Comment (cconard96) [reply source/modules/configuration/notifications/template_example.rst:217]: Oops, I forgot the language dropdown isn't localized.

**Review by trasher — APPROVED** [review]

**Pull request merged**